### ISSUE TITLE
Wayyiqtol text 

### ISSUE THREAD

**Issue by jloewenstein** [EzerIT/BibleOL#59]
I recently experimented it an exercise for parsing verbs in a passage. 
I chose the passage, then I asked for all the verbs, and asked the the text be shown, and the gender person number ect. be requested. 
The exercise works perfectly. What is somewhat of an issue is that I noticed the text output is missing the ו of ויקטול:
![Screenshot 2024-03-20 122853](https://github.com/EzerIT/BibleOL/assets/163320105/1e811a31-996e-4803-9ef3-0eceecf8d8a1)
The marked 'word' in the text is just יראו, and the text out put matches, but the correct answer to tense is wayyiqtol - which the system knows is correct. 
This might be more of a debated linguistic issue, but I would much prefer the the student be shown the full 'word' including the prefixed ו, since the leftover form does not even always exist on its own. for example: the wayyiqtol of the root אמר is וַיֹּ֥אמֶר (Gen1:3)  the extracted text that would be shown will exclude the ו and look like this: יֹּ֥אמֶר  (which does not exist on its own ever) while the actual yiqtol form is: יֹאמַ֗ר (Geb31:8).

By the way, In my opinion, the infinitive construct like the one appearing in the image above: לְהַגִּיד should also be considered one 'word' to be marked and appear together in the text output

**Comment by tmccormack165** [comment]
Hi @jloewenstein, thank you for notifying me of that, right now I have not studied Hebrew yet (although I plan to), so I will bring this up with the team.

**Comment by oliverglanz** [comment]
@jloewenstein this is a known problem. It has to do with the fact that a wayyiqtol form is consisting of two monads (smallest units of the database (the conjucntion and the verbs). This allows you to find Waw conjunctions even if they are part of a wayyiqtol. The problem is, that the tense information (wayyiqtol) is stored with the verb. But since a morphological exercise is based on monads (words) it is not possible to the word text together with the Waw since that would actually be two words of which the first word (waw) does not have tense information. I am sure there is a way to change this, but it would require a bit of thinking. @tmccormack165 this should be something we can discuss in our next meeting. Again, I am willing to help...

**Comment by ernstboogert** [comment]
Yes, lets discuss this in our next meeting. Maybe we can use some hacks to display it correctly. As Oliver noted it is not possible to display it correctly straight from the database data. Also the infinitivus need to be considered. Tim and I will dig into it to find a convenient solution.

**Comment by jloewenstein** [comment]
Thank you!
I was guessing that this was it:)
May you have a great idea that can be implemented easily:)

**Comment by ernstboogert** [comment]
I think we can easily select the previous monad to colorcode when the verb is Wayyiqtol... Timothy will experiment with it...

**Comment by oliverglanz** [comment]
@ernstboogert @tmccormack165 , If you want I can also create a new feature for all words that shows the text features except its a wayyiqtols or infintiives form where it will show the text features + the text feature of the previous word.
